result/nd.yaml

```diff
@@ -0,0 +1,73 @@
+openapi: 3.1.0
+
+info:
+  title: Neighbor Discovery table models
+  version: ^0.0.0
+
+components:
+  schemas:
+    Nd.Entries.Request:
+      description: >-
+        The request to retrieve Neighbor Discovery entries per IPv6 interface.
+      type: object
+      properties:
+        ipv6_names:
+          description: >-
+            The names of IPv6 interfaces for which ND entries will be retrieved.
+            If no interface names are specified then the result set will contain all ND entries.
```

Comment:
If no interface names are specified then the result set will contain all ND entries for all available interfaces.

Reply:
Description updated.